outlook/src/taskpane/components/Login/Login.tsx

```diff
@@ -136,8 +240,55 @@ class Login extends React.Component<{}, LoginState> {
                     <div>Search and store insights on your contacts.</div>
                 </div>
             </div>
-        </>
-    );
+        </>);
+    }
+
+    /**
+     * Check the "urlError" state and return the appropriate error message.
+     */
+    _getErrorMessage = () => {
+        switch (this.state.urlError) {
+            case UrlError.InvalidScheme:
+                return  (
+                    <span className="error-text">
+                        The database URL is invalid. It should follow the scheme "https://mycompany.odoo.com"
+                    </span>
+                );
+            case UrlError.DatabaseNotReachable:
+                return (
+                    <span className="error-text">
+                        We haven't been able to connect to your database.
+                        Check our <a href="https://www.odoo.com/documentation/14.0/applications/sales/crm/optimize/outlook_extension.html" target="_blank">FAQ</a>
```

Comment:
Shouldn't we rather refer to:
https://www.odoo.com/documentation/master/applications/sales/crm/optimize/outlook_extension.html

Instead of hardcoding the "14.0" ?
Maybe something to double check with Luc.